apps/schematic/api/schematic_api/controllers/utils.py

```diff
@@ -8,17 +8,40 @@
 import os
 import io
 import json
+import logging
+import subprocess
+from datetime import datetime, timedelta
+import re
+from math import ceil
+import yaml
 
 import pandas as pd
 from flask import request  # type: ignore
 from synapseclient.core.exceptions import (  # type: ignore
     SynapseNoCredentialsError,
     SynapseAuthenticationError,
 )
+from schematic.store import SynapseStorage
 from schematic.exceptions import AccessCredentialsError  # type: ignore
 
 from schematic_api.models.basic_error import BasicError
 
+# Config for various settable global values
+# Will use config.yaml if it exists, otherwise uses the example file
+# config.yaml is ignored by git so can be changed locally without accidentaly commiting it
+# To do so copy example_cofnig.yaml to config.yaml and make changes there
+if os.path.exists("config.yaml"):
+    with open("config.yaml", "r", encoding="utf-8") as file:
+        API_CONFIG = yaml.safe_load(file)
+else:
+    with open("example_config.yaml", "r", encoding="utf-8") as file:
```

Comment:
this here should be "default_config.yaml" instead of "example_config.yaml"?

Reply:
OH yep, for soem reaosn that change didn't get commited